pipeline/src/main/java/edu/rosehulman/aixprize/pipeline/core/Controller.java

```diff
@@ -0,0 +1,19 @@
+package edu.rosehulman.aixprize.pipeline.core;
+
+import org.apache.commons.logging.Log;
+import org.ros.namespace.GraphName;
+import org.ros.node.*;
+
+public class Controller extends AbstractNodeMain {
+
+	@Override
+	public GraphName getDefaultNodeName() {
+		return GraphName.of("pipeline/core");
+	}
+	
+	@Override
+	public void onStart(ConnectedNode connectedNode) {
+		Log log = connectedNode.getLog();
+		log.info("Hello world!");
```

Comment:
Hey, I understand this part!

Reply:
That's because it's the only part I actually wrote (besides the README.org).